integration/multiarch/patch_test.go

```diff
@@ -126,6 +126,115 @@ func TestPatch(t *testing.T) {
 	}
 }
 
+// Tests patching only some architectures while preserving others.
+func TestPatchPartialArchitectures(t *testing.T) {
+	// Test image with multiple platforms including Windows
+	originalImage := "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver"
```

Comment:
can we add this to retag using registry.k8s.io/csi-secrets-store/driver

Reply:
looks like `registry.k8s.io/csi-secrets-store/driver:v1.4.8` exists, changed it to that. I'm not sure what you mean by add to retag?